refactoring

vovaaa312 opened the pull request:
Program.cs was split into several classes, separating logic into different units to improve maintainability (Mathematics: Poly, Cplx; Rendering: FractalRenderer; Utils: ArgumentParser)

vovaaa312 committed:
refactoring

rodi0878 commented on NNPTPZ1/Mathematics/Cplx.cs:
```diff
@@ -0,0 +1,84 @@
+                return base.Equals(obj);
+            }
+
+            public readonly static Cplx Zero = new Cplx()
```
public constants and properties should be at the top of class

rodi0878 commented on NNPTPZ1/Mathematics/Cplx.cs:
```diff
@@ -0,0 +1,84 @@
+                    Imaginari = (float)(a.Re * b.Imaginari + a.Imaginari * b.Re)
+                };
+            }
+            public double GetAbS()
```
wrong method name

rodi0878 commented on NNPTPZ1/Mathematics/Cplx.cs:
```diff
@@ -0,0 +1,84 @@
+                    Imaginari = a.Imaginari + b.Imaginari
+                };
+            }
+            public double GetAngleInDegrees()
```
method name doesnt reflect code

rodi0878 commented on NNPTPZ1/Mathematics/Poly.cs:
```diff
@@ -0,0 +1,99 @@
+            /// <summary>
+            /// Coe
+            /// </summary>
+            public List<Cplx> Coe { get; set; }
```
whats Coe?

rodi0878 commented on NNPTPZ1/Mathematics/Poly.cs:
```diff
@@ -0,0 +1,99 @@
+            public Poly Derive()
+            {
+                Poly p = new Poly();
+                for (int q = 1; q < Coe.Count; q++)
```
q? p?

rodi0878 reviewed (changes requested):
Files Poly and Cplx needs more refactoring

vovaaa312 committed:
update:
Cplx.cs: public constants and properties, GetAbs and GetAngleInDegrees method name fix
Poly.cs: variables naming fix

vovaaa312 commented:
update:
Cplx.cs: public constants and properties, GetAbs and GetAngleInDegrees method name fix
Poly.cs: variables naming fix

rodi0878 commented:
ok accepted
